### PR TITLE
Website Terms and Conditions

### PR DESCRIPTION
**Pull request by ajcumine**
### Changes
Add Website Terms page at `/terms-and-conditions`
Update Privacy and Cookies policies with links to Terms

#### Trello Ticket / Github Issue Link
https://trello.com/c/lphTgz96/219-red-badger-website-terms

### CODE REVIEW

File: site/pages/terms-and-conditions/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <Policy
+      title="Website Terms"
+      effectiveDate="6 of June 2018"
+      policies={policies}
+      social={social}
+    />
+  );
+};
+
+export default CookiePolicyPage;
```
Comment (dpiatek) [inline site/pages/terms-and-conditions/index.js]: `TermsAndConditionsPage` ?

**Comment by dpiatek** [comment]
@ajcumine think we don't link to this from the homepage. Should we ask Andrea or Nathalie for new copy that could go into the footer (where `We use cookies on our website. For more information, view our privacy policy.` now is)?

File: site/pages/terms-and-conditions/index.js (hunk 1 above)
Comment (ajcumine) [reply site/pages/terms-and-conditions/index.js]: derp

**Comment by ajcumine** [comment]
Yeah will do tomorrow when they're back in 👍

**Pull request merged**